Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Alpine por que tem python 3.9: https://hub.docker.com/_/python
+FROM python:3.9-rc-alpine
```

Comment:
Nunca trabalhei com versões `rc` (_release candidate_) a não ser quando quero explicitamente testar se a aplicação vai funcionar na próxima versão do Python. _Release candidates_ podem ainda ter bugs, o que pode atrapalhar o processo de desenvolvimento, especialmente para pessoas não tão calejadas no Python.

Por isso, se não for um requisito o Python 3.9 eu ficaria com o Python 3.8 até o 3.9 ser oficialmente lançado como _estável_.

Reply:
Beleza, eu acho que faz sentido usar o 3.8 e uso slim daí! Valeu cudos!